components/function-controller/pkg/apis/serverless/v1alpha1/function_defaults_test.go

```diff
@@ -18,6 +20,12 @@ func TestSetDefaults(t *testing.T) {
 	zero := int32(0)
 	one := int32(1)
 	two := int32(2)
+
+	functionProfiles := `
+{
+	"python39": "L"
```

Comment:
As agreed offline we need to align values with values in sub-chart and chart.

Reply:
I clean up the charts.